roles/gluster-centos-repo/vars/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+gluster_centos_repos:
+    nightly:
+        description: "Gluster nightly builds from CentOS Storage SIG"
+        baseurl: "http://artifacts.ci.centos.org/gluster/nightly/master/7/x86_64/"
+    3-12:
+        description: "Gluster 3.12 from CentOS Storage SIG"
+        baseurl: "http://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.12/"
+    3-11:
+        description: "Gluster 3.11 from CentOS Storage SIG"
+        baseurl: "http://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.11/"
+    3-10:
+        description: "Gluster 3.10 from CentOS Storage SIG"
+        baseurl: "http://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.10/"
+    3-9:
```

Comment:
We probably don't need this many repositories and we should stick with nightly, 3-12 and maybe 3-11 only.

Reply:
That's true, we need to verify which versions should be available for testing in upstream and remove the others ...